client/signin.js

```diff
@@ -0,0 +1,25 @@
+const button = document.getElementById('fit-footer');
+button.addEventListener('click', (event) => {
+  const emailInput = document.getElementById('email');
+  const passwordInput = document.getElementById('password');
+  const messageText = document.querySelector('p');
+  const inputData = {
+    email: emailInput.value,
+    password: passwordInput.value
+  };
+  event.preventDefault();
+  fetch('https://m-tracker.herokuapp.com/api/v1/auth/login', {
+    method: 'POST',
+    headers: { 'content-type': 'application/json' },
+    body: JSON.stringify(inputData)
+  })
+    .then((response) => { return response.json(); })
+    .then((data) => {
+      if (data.message === 'Login successful') {
+        window.location.href = 'index.html';
+        sessionStorage.setItem('token', data.token);
```

Comment:
why are you using sessionStorage being used here?

Reply:
No special reason, do you suggest I use another ? will localStorage do ?

Reply:
Go and research the difference between localStorage and sessionStorage there is a use case for both

Reply:
alright